models/Room.js

```diff
@@ -0,0 +1,27 @@
+import mongoose from 'mongoose';
+
+const RoomSchema = mongoose.Schema(
```

Comment:
@jimmyygit we should follow a standard convention before this becomes an issue. Can we call our "rooms", "spaces" from now on. It adheres to our brand a little more.

Reply:
should have 3 more fields.
1. spaceId (_id is reserved by mongoose so it's better to have a clear separation in the namespace)
2. participants
3. isActive